bindgen/templates/RecordTemplate.cs

```diff
@@ -6,6 +6,17 @@
 {%- let (ordered_fields, is_reordered) = rec.fields()|order_fields %}
 
 {%- call cs::docstring(rec, 0) %}
+{%- for field in ordered_fields %}
+{%- match field.docstring() %}
+{%- when Some with(docstring) %}
+/// <param name="{{ field.name() }}">
+{%- for line in docstring.lines() %}
+/// {{ textwrap::dedent(line) }}
+{%- endfor %}/// </param>
```

Comment:
Isn't trailing `</param>` supposed to be on a new line?
```suggestion
{%- endfor %}
/// </param>
```

Reply:
No. It looks funky here, but I had to make this change or else an empty line is actually emitted in the resulting file before the `/// </param>` line. If there's a better way to avoid an empty line, let's do it.

Reply:
:thinking: I'm getting a different result. With trailing `</param>`.
```
dictionary RecordTest {
    /// This is a description for record field.
    /// A second line describing docstring field.
    i32 test;

    /// This is a description for record field.
    /// A second line describing docstring field.
    i32 test2;
};
```

```cs
/// <param name="test">
/// This is a description for record field.
/// A second line describing docstring field./// </param>
/// <param name="test2">
/// This is a description for record field.
/// A second line describing docstring field./// </param>
internal record RecordTest(int @test, int @test2) { }
```

Reply:
Are you running on Windows or some other OS? I was testing on Windows. Any chance that is related?

Reply:
Yes, you are right. I tried on Windows, and indeed there is a blank line. It seems like the blank line comes from `docstring.lines()`, where the last line contains new line character for some reason. I'm not entirely sure where this new line character comes from🤔 Looks like there is a general docstring parsing issue in uniffi-rs upstream.

Trimming whitespace from each line helps.
```
/// {{ line.trim_end() }}
```

This is the full version I tested right now.
```
{%- call cs::docstring(rec, 0) %}
{%- for field in ordered_fields %}
{%- match field.docstring() %}
{%- when Some with(docstring) %}
/// <param name="{{ field.name() }}">
{%- let docstring = textwrap::dedent(docstring) %}
{%- for line in docstring.lines() %}
/// {{ line.trim_end() }}
{%- endfor %}
/// </param>
{%- else %}
{%- endmatch %}
{%- endfor %}
```